src/app.jsx

```diff
@@ -545,11 +547,28 @@ class Application extends React.Component {
                     }
                 });
 
+        cockpit.spawn(["systemctl", "--user", "show", "--value", "-p", "LoadState", "podman-restart"], { environ: ["LC_ALL=C"], error: "ignore" })
```

Comment:
How would this look as a d-bus query? calling systemctl is very heavy compared to a d-bus call.

Reply:
Hmm we first have to look up the unit path and then get the LoadState property. I am not entirely sure how this works for a user session.

Reply:
This will be called even when root logs in. I think it makes more sense in `checkUserService`?

Reply:
Fixed.